omero/developers/Web/Deployment.txt

```diff
@@ -11,8 +11,8 @@ external OMERO server. You can add the server to the
 You should enable :property:`omero.web.debug` and start a lightweight
 development Web server on your local machine.
 
-.. note:: Since OMERO 5.2, the web framework is no longer shipped with the
-    Django package and requires manual installation of the Django dependency.
+.. note:: Since OMERO 5.2, the OMERO web framework no longer bundles a copy
+    of Django package and requires manual installation of the Django dependency.
```

Comment:
I think the whole sentence might be better as something like:

> Since OMERO 5.2, the OMERO web framework no longer bundles a copy
> of the Django package,  instead manual installation of the Django dependency
> is required.

in both cases.